ddtrace/llmobs/_llmobs.py

```diff
@@ -105,16 +106,20 @@ def __init__(self, tracer=None):
         self.tracer = tracer or ddtrace.tracer
         self._llmobs_context_provider = LLMObsContextProvider()
         self._llmobs_span_writer = LLMObsSpanWriter(
-            is_agentless=config._llmobs_agentless_enabled,
-            agentless_url="%s.%s" % (AGENTLESS_BASE_URL, config._dd_site),
+            site=config._dd_site,
+            api_key=config._dd_api_key,
             interval=float(os.getenv("_DD_LLMOBS_WRITER_INTERVAL", 1.0)),
             timeout=float(os.getenv("_DD_LLMOBS_WRITER_TIMEOUT", 5.0)),
+            is_agentless=config._llmobs_agentless_enabled,
+            _agentless_url="%s.%s" % (AGENTLESS_SPAN_BASE_URL, config._dd_site),
```

Comment:
Realistically the only time we manually set `_agentless_url` is in testing when we want to set the localhost test backend as the url. Will move the `_agentless_url` value to be set inside the writer itself and only provide this to override (if provided).